src/orderbook.js

```diff
@@ -106,7 +140,16 @@ function OrderBook(remote,
       switch (action) {
         case 'add':
           if (++self._listeners === 1) {
+            self._shouldSubscribe = true;
             self.subscribe();
+
+            self._remote.on('transaction', updateFundedAmountsWrapper);
+            self._remote.once('disconnect', onDisconnect);
+
+            if (self._isAutobridgeable) {
```

Comment:
Adding the model's here would mean that calling the autobridged books's subscribe method would not result in subscription to the leg books - only the 'auto subscribe' method of adding a listener or its initial creation would result in a proper subscription (not a problem for trade since it subscribes by adding a listener)